### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by longyanrou** [mileyan/AnyNet#1]
Hi,
Good job!
I used your code to train the network, but found that the network converges too slow. I got "Average train loss = Stage 0 = 7.32	Stage 1 = 6.86	Stage 2 = 6.38	Stage 3 = 5.62" after 8 epoch. Is it normal? And could you please release the trained model?
Besides, you compared your network with StereoNet. I tried to re-implement StereoNet using PyTorch, but the runtime is much longer than that reported in the paper. I got about 0.17s when testing images from Scene Flow dataset on 1080 ti GPU. I wonder if you can achieve the runtime reported in StereoNet using your re-implementation. I will be very grateful if you could share the implementation of StereoNet too.
Thank you very much.

**Comment by longyanrou** [comment]
The pretraining on Scene Flow dataset （10 epoches） took me 17.68 Hours on two 1080 ti GPU cards. And I achieved "Average test EPE = Stage 0=5.24, Stage 1=4.77, Stage 2=4.30, Stage 3=3.58".

**Comment by longyanrou** [comment]
Besides, there is another question. It seems to exist inconsistency between the network architecture described in TABLE 1 and the code.

The first line of Regularization part in TABLE 1, it says
`
23-27 | 3×3×3 3-D conv with 16 filters
`
But in the submodules.py file, according to the function post_3dconvs:
`def post_3dconvs(layers, channels):
    net  = [batch_relu_conv3d(1, channels)]
    net += [batch_relu_conv3d(channels, channels) for _ in range(layers)]
    net += [batch_relu_conv3d(channels, 1)]
    return nn.Sequential(*net)
`
there are 4 layers.

Maybe the code should be revised to:
`def post_3dconvs(channels, layers):
    net  = [batch_relu_conv3d(1, channels)]
    net += [batch_relu_conv3d(channels, channels) for _ in range(layers)]
    net += [batch_relu_conv3d(channels, 1)]
    return nn.Sequential(*net)
`
?

Thanks.

**Comment by longyanrou** [comment]
Besides, there is another question. It seems to exist  inconsistency between the network architecture described in TABLE 1 and  the code. The first line of Regularization part in TABLE 1, it says 
```
23-27 | 3×3×3 3-D conv with 16 filters 
```
But in the submodules.py file, according to the function post_3dconvs:
```
def post_3dconvs(layers, channels): 
    net  = [batch_relu_conv3d(1, channels)] 
    net += [batch_relu_conv3d(channels, channels) for _ in range(layers)]
    net += [batch_relu_conv3d(channels, 1)] 
    return nn.Sequential(*net) 
```
 there are 4 layers.

Maybe the code should be revised to: 
```
def post_3dconvs(channels, layers):
     net  = [batch_relu_conv3d(1, channels)]
     net += [batch_relu_conv3d(channels, channels) for _ in range(layers)]
     net += [batch_relu_conv3d(channels, 1)]
     return nn.Sequential(*net) 
```
 ? 

Thanks.
